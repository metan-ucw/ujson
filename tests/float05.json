{"val": 1.1e-1}
