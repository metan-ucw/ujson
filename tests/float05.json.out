{
 val: 0.110000
}
